CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/item/MCItemStack.java

```diff
@@ -106,7 +106,16 @@ public static MCItemStack createNonCopy(ItemStack itemStack) {
     public IItemDefinition getDefinition() {
         return new MCItemDefinition(stack.getItem().getRegistryName().toString(), stack.getItem());
     }
-    
+
+    @Override
+    public boolean isItemBlock() {
+        if (stack.isEmpty()) {
```

Comment:
Technically, that should never happen, but why not ^^

Reply:
I've gotten used to being super-defensive when it comes to coding. I'm happy to remove the check, especially as it's more likely that other things will fail/crash in the event of it being empty.

Reply:
I would remove it, just because the other methods don't have it